Terminal.Gui/Drivers/ConsoleDriver.cs

```diff
@@ -443,6 +443,10 @@ public abstract class ConsoleDriver {
 		/// </summary>
 		public abstract int Rows { get; }
 		/// <summary>
+		/// If true, it allows to write at the beginning of the next line if it's exceed the column limit.
+		/// </summary>
+		public abstract bool AllowNewLine { get; set; }
+		/// <summary>
```

Comment:
I do not think that we should have this, if we did, it should be called "Auto-wrap", but this really belongs as a higher-level concept.

Instead, if we are out of boundary, this should cut the text.

If a view has a desire to wrap around, it should do this on its own, this should not be magic in the driver, otherwise, well intended code with bugs will have additional bugs to work around.

Reply:
That it's what is it is actually doing. It's only let drawn if it isn't not out of boundary or only let drawn if the AllowNewLine is defined.

Reply:
Done

Reply:
> If a view has a desire to wrap around, it should do this on its own, this should not be magic in the driver, otherwise, well intended code with bugs will have additional bugs to work around.

All views can write directly to the console driver, which would require having to redraw all views to obtain the same result. So managing this directly in the driver avoids all that work that could even be useless, overwhelming and possibly not have the desired result.